gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2

```diff
@@ -1876,3 +1876,34 @@ def test_{{ method_name }}_empty_call():
         {% endwith %}{# method_settings #}
         assert args[0] == {{ method.input.ident }}()
 {% endmacro %}
+
+
+{% macro transport_kind_test(service, opts, is_async=False) %}
+@pytest.mark.parametrize("transport_name", [
```

Comment:
Good suggestion but I'd like to punt this until I add an entry for `rest_asyncio` and see how the changes turn out. I will be refactoring more unit tests as I go so this won't be ignored! Let me know if you are okay with it!

Reply:
OK, that's fine. I think it will make the code clearer, so the sooner we do it the better, but if you want to wait until one of the upcoming CLs, that's fine too.